romancal/resample/resample.py

```diff
@@ -335,6 +338,12 @@ def resample_many_to_one(self):
 
         return self.output_models
 
+    # TODO Rename this here and in `resample_many_to_many` and `resample_many_to_one`
```

Comment:
If I understand correctly, you already did this #TODO, and could remove it now?

Reply:
Thanks for catching that, Eddie!